lib/tasks/import.rake

```diff
@@ -0,0 +1,56 @@
+require 'csv'
+
+namespace :import do
+  desc "import CSV data into database"
+  task projects: :environment do
+    csv = File.open('lib/data/seed/collated-coral.csv')
+
+    csv_headers = File.readlines(csv).first.split(",")
+
+    title_hash = {
```

Comment:
I think it would be better to rename the hash as `project_hash`, as you are talking about project data in this case